packages/chaire-lib-frontend/src/components/forms/auth/anonymous/AnonymousLogin.tsx

```diff
@@ -1,9 +1,14 @@
+/*
+ * Copyright 2022, Polytechnique Montreal and contributors
```

Comment:
Is 2022 the right date?

Reply:
yes, the header was missing, but the file is from 2022.